### DISCUSSION TITLE
Inspiração para Blog Pessoal

### DISCUSSION THREAD

**Discussion by fabiomirandaa** [frontendbr/forum#1541] [Discussão]
Recentemente aqui no Fórum houveram várias discussões sobre como criar Blogs(qual tecnologia usar), e até sobre questões mais morais como o uso do Medium estar relacionado a algumas questões. 

Mas, uma coisa eu não vi e eu tenho uma certa dificuldade nessa questão: Qual é a inspiração de vocês para fazer o layout do site pessoal ou blog? Em que se inspiram? Seguem pessoas? Misturam características de sites famosos? Como fazer um layout e largar a dependência de só catar um tema e modificar pequenas coisas e sair usando?

**Comment by wilsonneto-swe** [comment]
Opa, bom dua fabio!

Eu tenho uma pasta de favoritos onde salvo todos os sites bonitos que vejo, acesso sempre o awwwards e o siteinspire (diariamente), para sempre estar vendo site bonitos e bem feitos, assim quando pego um projeto novo já tenho em mente algumas coisas bem legais que vi e quero colocar em prática.

Abraços!

**Comment by iagobruno** [comment]
Pra criar o design de um site geralmente eu procuro inspiração no [Dribbble](https://dribbble.com), [Behance](https://www.behance.net), no [Pinterest](https://pinterest.com) e me inspiro no layout de outros projetos.
Mas pra blog não tem segredo cara, é uma página com uma lista de postagens e a página individual pra cada postagem com título, conteúdo e umas informações adicionais ai você usa sua criatividade artística seguindo essa estrutura.

**Comment by felipefialho** [comment]
No meu caso, entro em vários blogs de desenvolvimento e vou anotando o que acho bom em cada um. Isso falando de layout. 

Apesar de que o layout do [meu site](http://felipefialho.com) não mudou tanto com relação a versão de 2015, fiz apenas ajustes pontuais para trazer o layout para uma cara mais 2019. Na essência ele já estava legal.

O ponto que mais foquei foram os textos, foram horas estudando quais as melhores formas de renderizar fontes de sistema e resultar numa leitura fácil. Fiquei satisfeito com o resultado.

**Comment by deppbrazil** [comment]
Ainda não desenvolvi o meu pessoal, mas sigo alguns que acho o layout muito foda e servem como inspiração, 

São esse: 
* [Willian Justen](https://willianjusten.com.br/)
* [Raphael Fabeni](https://raphaelfabeni.com/blog/)
* [Felipe Fialho](https://www.felipefialho.com/blog/)

**Comment by diessica** [comment]
tem o meu mas é meio emo às vezes https://diessi.ca

**Comment by felipefialho** [comment]
Gosto muito do layout do seu @diessica, minimalista e bonito 😄
